### DISCUSSION TITLE
同样的代码,每次的日志顺序是不一样的

### DISCUSSION THREAD

**Discussion by nightwhite** [labring/laf#678] [新春贺岁版反馈专区]
![image](https://user-images.githubusercontent.com/22722256/214103659-a863f6d3-c078-4f15-91ae-4ae4a976ccc6.png)
![image](https://user-images.githubusercontent.com/22722256/214103791-b9fb4b0d-0ba6-4550-8058-a9d614049e89.png)

同样的代码,每次的日志顺序是不一样的,并且顺序完全随机

**Comment by LeezQ** [comment]
1. 前端先做一次 reverse()
2. 已知问题，待处理，原因是由于日志存入的时候是异步的，所以顺序不可控。需要优化
